### PR TITLE
Mobile contact us page styling

### PR DESCRIPTION
**Pull request by Ajknight121**
resolves issue #108, #102, #101, #107, #104

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -50,4 +50,12 @@ h5 {
...
 input[type = 'text'] {
   @apply text-xl rounded-3xl border-2 border-[#AAAAAA] bg-[#EEEEEE] w-full px-4 py-1;
+}
+
+textarea[type = 'textarea'] {
+  @apply text-xl rounded-3xl border-2 border-[#AAAAAA] bg-[#EEEEEE] w-full px-4 py-1;
+}
+
+select{
+  @apply text-xl rounded-3xl border-2 border-[#AAAAAA] bg-[#EEEEEE] w-full px-4 py-1;
```
Comment (ChristianBingman) [inline src/index.css:60]: This seems to be broken on safari, does it work on yours?

**Review by ChristianBingman — CHANGES_REQUESTED** [review]
Just need you to attach all relevant issues, and fix the select box if possible. Thanks and sorry for the delay :)

**Comment by ChristianBingman** [comment]
Also, please resolve the merge conflict

File: src/index.css (hunk 1 above)
Comment (Ajknight121) [reply src/index.css:60]: The styling works on chrome for me.

Comment (ChristianBingman) [reply src/index.css:60]: Try this https://stackoverflow.com/questions/7638677/how-can-i-remove-the-gloss-on-a-select-element-in-safari-on-mac

File: src/index.css [hunk 2]
```diff
@@ -50,4 +50,14 @@ h5 {
...
```
Comment (ChristianBingman) [inline src/index.css:60]: Since the styling was removed, you need to re add the dropdown arrow.

**Review by ChristianBingman — CHANGES_REQUESTED** [review]

**Pull request closed without merging**